src/App.jsx

```diff
@@ -1,41 +1,70 @@
-import React from 'react';
-import './App.scss';
-import './styles/general.scss';
+import React, { useState } from 'react';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
+import { UserSelect } from './components/UserSelect';
+import { getPostDetails, getPostComments, deleteComment } from './api/posts';
+import './App.scss';
+import './styles/general.scss';
+
+const App = () => {
+  const [person, setPerson] = useState({});
+  const [postId, setPostId] = useState();
+  const [post, setPost] = useState(null);
+  const [isChoosen, setIsChoosen] = useState(false);
+  const [comments, setComments] = useState([]);
+
+  const select = (user) => {
+    setPerson(user);
+  };
+
+  const loadComments = (id) => {
+    getPostComments(id)
+      .then(result => setComments(result));
+  };
+
+  const selectedPostId = (id, buttonStatus) => {
+
+    setIsChoosen(buttonStatus);
+    setPostId(id);
+
+    getPostDetails(id)
+      .then(result => setPost(result.data));
+
+    loadComments(id);
+  };
```

Comment:
selectedPostId? what does it mean? it's a function, not a value. Name it properly, so it would be understandable what it does